books-i-read.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My Curriculum Vitae</title>

```

Comment:
Change this title to be accurate for the content. "My recommended reading" or "favorite books"